e2e/chat.spec.ts

```diff
@@ -86,6 +94,8 @@ describe('chat', () => {
     const chat = new Chat(page);
     await chat.goto();
 
+    await chat.switchToBlockchainModel();
+
```

Comment:
However, we will keep it around for tests that do blockchain queries and transactions.